docs/extensions/overview.md

```diff
@@ -6,53 +6,24 @@ description: >-
 ---
 
 import IdeList from "/src/components/reference/_ide-list.md"
-import Login from "/src/components/procedure/_login-activate.mdx"
 
 # Extensions
 
 Several third-party tools include Semgrep extensions.
 
-### Official IDE extensions
+## Official IDE extensions
 
 <IdeList />
 
-### Use of Language Server Protocol (LSP)
+## Use of Language Server Protocol (LSP)
 
-All of the official extensions use the [Language Server Protocol](https://microsoft.github.io/language-server-protocol/) to communicate with Semgrep. This allows the team to focus on one codebase that can be shared across most modern editor platforms.
+All of the official IDE extensions use the [Language Server Protocol](https://microsoft.github.io/language-server-protocol/) to communicate with Semgrep. This allows the team to focus on one codebase that can be shared across most modern editor platforms.
 
-### Pre-commit
+## `pre-commit`
 
-The [pre-commit framework](https://pre-commit.com/) can run `semgrep` at commit-time. [Install `pre-commit`](https://pre-commit.com/#install) and add the following to `.pre-commit-config.yaml`
+Prevent secrets or security issues from entering your Git source control history by running Semgrep as a [<i class="fas fa-external-link fa-xs"></i> pre-commit](http://localhost:3000/docs/extensions/overview) hook. See [`pre-commit` documentation](/extensions/pre-commit) for details.
 
-```yaml
```

Comment:
@semgrep-shichao Can you confirm 2 things:

1. The sample here (`semgrep`) actually works as-is.
2. The sample for `semgrep ci` (logged in) did not work for your specific use case in slack?